loginas/utils.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from django.conf import settings as django_settings
+from django.contrib.auth.models import User
+from django.contrib.auth import load_backend, login, logout
+from django.contrib import messages
+from django.core.signing import TimestampSigner, SignatureExpired
+from datetime import timedelta
+
+from . import settings
```

Comment:
I agree, but it would be like calling a module `models` that had nothing to do with database models, and renaming the actual models to `django_models`. It's just surprising behavior, but I'm not that bothered by it, you can leave it if you like.